recipes/aws-libfabric/all/conanfile.py

```diff
@@ -0,0 +1,115 @@
+from conans import ConanFile, AutoToolsBuildEnvironment, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+
+required_conan_version = ">=1.35.0"
+
+class LibfabricConan(ConanFile):
+    name = "aws-libfabric"
+    description = "AWS Libfabric"
+    topics = ("fabric", "communication", "framework", "service")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/aws/libfabric"
+    license = "BSD-2-Clause", "GPL-2.0-or-later"
+    settings = "os", "arch", "compiler", "build_type"
+    _providers = ["gni", "psm", "psm2", "sockets", "rxm", "tcp", "udp", "usnic", "verbs", "bgq", "shm", "efa", "rxd", "mrail", "rstream", "perf", "hook_debug"]
+    options = {
+        **{ p: [True, False, "shared"] for p in _providers },
+        **{
+            "shared": [True, False],
+            "fPIC": [True, False],
+            "with_libnl": [True, False],
+            "with_bgq_progress": ["auto", "manual"],
+            "with_bgq_mr": ["basic", "scalable"]
+        }
+    }
+    default_options = {
+        **{ p: False for p in _providers if p not in ["efa", ] },
+        **{
+            "shared": False,
+            "fPIC": True,
+            "tcp": True,
+            "with_libnl": False,
+            "with_bgq_progress": "manual",
+            "with_bgq_mr": "basic"
+        }
+    }
+    build_requires = "autoconf/2.71", "automake/1.16.3", "libtool/2.4.6"
```

Comment:
For consistency with other recipes, please add a `build_requirements` method that only does `self.build_requires("libtool/2.4.6")`.
libtool internally depends on automake which depends on autoconf.
This way, when a new version of autoconf is released, we don't have to modify all recipes that need it indirectly.